modules/system/console/MixList.php

```diff
@@ -35,16 +34,13 @@ public function handle(): int
             return 0;
         }
 
-        $this->info('Packages registered:');
-        $this->line('');
-
         $errors = [];
 
         $rows = [];
         foreach ($packages as $name => $package) {
             $rows[] = [
                 'name' => $name,
-                'active' => $package['ignored'] ?? false ? '<fg=red>No</>' : '<info>Yes</info>',
+                'active' => $this->option('json') ? !$package['ignored'] : ($package['ignored'] ? '<fg=red>No</>' : '<info>Yes</info>'),
```

Comment:
I'm not a fan of double-ternary usage - it makes it hard to visually parse what is happening. Perhaps we just change this to a boolean, and we'll control the output below.

```suggestion
                'active' => !$package['ignored'],
```